steps/common.py

```diff
@@ -1,7 +1,117 @@
 '''Common test methods'''
 
+import os
+import re
+
 from behave import *
 
+
+def exec_service_cmd(context, action, service_name, host=None):
+    if host == None:
+        host = context.target_host
+
+    #cmd = context.service_cmd
+    cmd = "systemctl ACTION SERVICE_NAME"
+    cmd = re.sub("ACTION", action, cmd)
+    cmd = re.sub("SERVICE_NAME", service_name, cmd)
+
+    result = context.remote_cmd(cmd='command',
+                                host=host,
+                                module_args=cmd,
+                                ignore_rc=True)
```

Comment:
I considered to use service module in my test and use the exist functions in common.py, but I met foloowing problem:
service module request set at least one args from 'state' or 'enabled', so when we try to get the status of service we may change it. But in my test I want to get the service status without change it. 

Any suggestion about this?